CPAC/registration/registration.py

```diff
@@ -3218,14 +3219,20 @@ def coregistration_prep_mean(wf, cfg, strat_pool, pipe_num, opt=None):
         "input",
     ],
     option_val="fmriprep_reference",
-    inputs=["desc-ref_bold"],
-    outputs=["sbref"],
+    inputs=[
```

Comment:
This needs to take in `desc-unifized_bold` as an input, otherwise `strat_pool.check_rpool("desc-unifized_bold"):` will never be able to be true.